checkout_cache_criteria.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
We may not wish to begin collection configuration files in the repo.

Consider inlining this into `CHECKOUT_CACHE_CRITERIA` in environment for now. In the future we may need another strategy if the cache criteria become too large for an env var.

Reply:
moved to environment